nautobot/core/tests/test_graphql.py

```diff
@@ -1,36 +1,29 @@
 import types
+import uuid
+
+# from parameterized import parameterized
 
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import Group
 from django.contrib.contenttypes.models import ContentType
-from django.test import TestCase
-from django.test import override_settings
+from django.test import TestCase, override_settings
+from django.test.client import RequestFactory
 from django.urls import reverse
 from graphene_django import DjangoObjectType
-from graphql.error import GraphQLLocatedError
+from graphene_django.settings import graphene_settings
+from graphql.error.located_error import GraphQLLocatedError
+from graphql import get_default_backend
 from rest_framework import status
 from rest_framework.test import APIClient
 
-from nautobot.utilities.testing import APITestCase
+
 from nautobot.users.models import ObjectPermission, Token
-from nautobot.dcim.models import (
-    Device,
-    Site,
-    Region,
-    Rack,
-    Manufacturer,
-    DeviceType,
-    DeviceRole,
-)
+from nautobot.dcim.models import Device, Site, Region, Rack, Manufacturer, DeviceType, DeviceRole, Interface
```

Comment:
I know some of the ordering in the `nautobot*` imports was already like this, but while you're here can you please sort lines 20-39 alphabetically? Thank you.